src/content/en/2020/capabilities.md

```diff
@@ -0,0 +1,459 @@
+these issues: For instance, new interfaces require the website to be sent over a secure connection
+(HTTPS). Some of them require a user gesture, such as a click or key press, to prevent fraud. Other
+capabilities require explicit permission by the user. Developers can use all APIs as a progressive
+enhancement: By feature detecting the APIs, applications won't break in browsers lacking support for
+those capabilities. In browsers that support them, users can get a better user experience. This way,
+web apps [progressively enhance](https://web.dev/progressively-enhance-your-pwa/) according to the
+user's particular browser.
+
+This chapter gives an overview of various modern web APIs, and the state of web capabilities in 2020
+based on usage data by the HTTP Archive. Due to technical limitations, the HTTP Archive only has
+data available for APIs that require neither permission, nor a user gesture. Where no data is
+available, the percentage of page loads in Google Chrome according to
+[Chrome Platform Status](https://chromestatus.com/metrics/feature/timeline/popularity) will be shown
+instead. Since some interfaces are brand-new and their usage is still very low, the statistics are
+not necessarily meaningful. However, in many cases trends can still be read from the data. Unless
+otherwise noted, the APIs are only available in Chromium-based browsers, and their specifications
+are in the early stages of standardization.
+
+## Async Clipboard API
+
+With the help of the `document.execCommand()` method, websites could already access the user's
+clipboard. However, this approach is somewhat restricted, as the API is synchronous (making it
+difficult to process clipboard items), and it can only interact with selected text in the DOM. This
+is where the [Async Clipboard API](https://web.dev/async-clipboard/)
+([W3C Working Draft](https://www.w3.org/TR/clipboard-apis/#async-clipboard-api)) comes in. This new
+API is not only asynchronous, meaning it doesn't block the page for large chunks of data or waiting
+for a permission to be granted, but also allows for images to be copied to or pasted from the
+clipboard in supported browsers such as Chrome, Edge, and Safari.
+
+### Read Access
+
+The Async Clipboard API provides two methods for reading content from the clipboard: A shorthand
+method for plain text, called `navigator.clipboard.readText()`, and a method for arbitrary data,
+called `navigator.clipboard.read()`. Currently, most browsers only support HTML content and PNG
+images as additional data formats. Due to privacy reasons, reading from the clipboard always
+requires the user's consent.
+
+(IMAGE: Async Clipboard API)
+
+The Async Clipboard API is comparatively new, so its usage is currently rather low. In March 2020,
+Safari added support for the Async Clipboard API in Safari 13.1. Over the course of 2020, the usage
+of the `read()` API was growing. In October 2020, the API was called during 0.0003 percent of all
```

Comment:
In general we do encourage percentage stats rather than absolute stats, however I wonder should we make an exception here? 0.0003 percent of sites is a difficult to comprehend number (except that it's really small!) so should we quantify this with an actual number:

```suggestion
of the `read()` API was growing. In October 2020, the API was called during 0.0003 percent of all page loads in Google Chrome (or approximately 20 pages).
```

Or something like that. What do you think @rviscomi ?

Reply:
If most of the stats are this small, I'd recommend a paragraph to set expectations with readers like:

> Many of the capabilities discussed in this chapter are still very new. So, unlike most chapters, stats will be presented as the absolute numbers of pages rather than relative percentages.

Reply:
@rviscomi Yes, most of the stats are this small. We need to check if we have absolute numbers for the Chrome Platform Status metrics available though.

Reply:
Sorry it's taken me so long to understand that these numbers are from Chrome Status and not HTTP Archive. That being the case, only relative stats are available.

For the HTTP Archive-based stats like StorageManager API :: Estimate, we should switch over from percent of pages to number of pages for consistency with the other HA-based stats that have even smaller adoption and need to use absolute page counts. I think it's ok for HA and Chrome-based stats to be different in terms of % of **page views** and number of **pages**.